quiche/src/recovery/rtt.rs

```diff
@@ -124,7 +132,19 @@ impl RttStats {
     }
 
     pub(crate) fn min_rtt(&self) -> Option<Duration> {
-        self.min_rtt.ne(&Duration::ZERO).then_some(*self.min_rtt)
```

Comment:
We are no longer initializing min_rtt to ZERO, so this check was not producing the intended result.  Looking at has_first_rtt_sample is more correct.

Reply:
verified that min_rtt updated here: https://github.com/cloudflare/quiche/commit/11e4e65041b533530cef5bc22a272f068be7fe78.

Ideally this would be a separate PR but separate commit works.